zh.json

```diff
@@ -190,11 +188,11 @@
 	"settings.clearTxAlert": "移除地址簿中所有地址!",
 	"settings.clearAddressBookAlert": "地址簿已全部清空!",
 	"settings.copySeedAlert": "种子短语已成功复制!",
-	"settings.removeWalletAlert": "复制并保存种子短语。否则你将永远失去钱包。您确认要删除钱包吗？",
+	"settings.removeWalletAlert": "复制并保存种子短语。否则你将永远失去钱包。您确认要删除钱包吗?",
```

Comment:
whats different?

Reply:
The difference between Chinese question mark and English question mark is not very important